packages/node-sdk/README.md

```diff
@@ -26,28 +26,25 @@ in **Bucket.co**.
 > information that is often sensitive and thus should not be used in
 > client-side applications.
 
+Create a `bucket.ts` file containing the following and set up the
+BUCKET_SECRET_KEY environment variable:
```

Comment:
what does this mean? does the `secretKey` automatically try to read from a `BUCKET_SECRET_KEY` env var? if so, I don't think that this is standard practise for 3rd party libs like ours. IMO it's better to be explicit with the passed `secretKey` config instead - all devs know how to pipe in an env var if they want to